packages/amplify-provider-awscloudformation/src/template-description-utils.ts

```diff
@@ -83,12 +83,15 @@ export function getDefaultTemplateDescription(context: $TSContext, category: str
   // get stack type "stackType"
   const stackTypeDescription = service ? `${category}-${service}` : category;
 
+  // get the reason why not switching to Gen2
+  const { whyContinueWithGen1 } = context.exeInfo.projectConfig;
```

Comment:
do you know if this is defined yet?

Reply:
It'd be defined in normal cases, but undefined when running `amplify export` if the project is created with `--yes` at `amplify init`.

Reply:
Fixed in [faa0859](https://github.com/aws-amplify/amplify-cli/pull/13849/commits/faa0859f81be5ef816e8a7f6e40bba88477390d0)